### PR TITLE
[IMP] theme_notes: update the theme

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Update the theme to improve the design with shapes, new pictures and new custom snippets.

**Improvements :**
- Update images
- Improve the typography (change Lobster by Montserrat)
- Change the tour
- Improve the use of the current palette color in snippets
- Change the header for the Minimalist

**Snippets improved :**
- Image_text
- Media_list
- Company_team
- Carousel

**New customisations :**
- Masonry_block
- Product_catalog
- Banner
- Cover
- Text_image
- Numbers
- Three_columns
- Color_blocks_2
- Image_gallery
- Call_to_action
- Features_grid
- Product_list
- Parallax
- Comparisons
- Quotes_carousel

Remove outdated theme description.

Task-2590454

### CODE REVIEW

File: theme_notes/static/description/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h2 class="text-muted">Html5 Responsive Bootstrap Theme for Odoo CMS</h2>
         first projects on the web. Just drag &amp; drop the new building blocks
         you need to easily build your layout. All layouts are made of sections,
         that you can easily combine to fit your specific project.</p>
-    <img src="Notes_description.png" class="img-fluid d-block mx-auto"/>
+    <img src="notes_description.jpg" class="img-fluid d-block mx-auto"/>
```
Comment (Cocographique) [inline theme_notes/static/description/index.html]: Move the changes related to the description page into the other commit.

File: theme_notes/static/src/scss/primary_variables.scss [hunk 2]
```diff
@@ -61,6 +61,8 @@ $o-theme-h4-font-size-multiplier: (24 / 16);
 $o-theme-h5-font-size-multiplier: (20 / 16);
 $o-theme-h6-font-size-multiplier: 1;
 
+$headings-font-weight: 700;
```
Comment (Cocographique) [inline theme_notes/static/src/scss/primary_variables.scss:64]: In bootstrap_overriden.scss:
```scss
$headings-font-weight: $o-theme-headings-font-weight !default;
```
In primary_variables.scss:
```scss
$o-theme-headings-font-weight: 700;
```

**Review by Cocographique — COMMENTED** [review]
As discussed before, try to reduce the number of customizations that do appear on the tour page.

**Comment by Brieuc-brd** [comment]
Merged here : https://github.com/odoo/design-themes/pull/500

**Pull request closed without merging**